docs/config/packages/framework.yaml

```diff
@@ -7,13 +7,11 @@ api_platform:
     json: ['application/json']
   docs_formats:
     jsonopenapi: ['application/vnd.openapi+json']
-  keep_legacy_inflector: false
```

Comment:
there still exists a bug which this flag is needed to workaround:
https://github.com/api-platform/core/issues/5934

shall we fix this first, before removing the workaround?

Reply:
https://github.com/api-platform/core/pull/6447 should fix this no?

Reply:
Thanks. Will give it a go